conda_forge_feedstock_ops/container_utils.py

```diff
@@ -140,55 +284,82 @@ def run_container_operation(
     data : dict-like
         The result of the operation.
     """
-    if mount_dir is not None:
-        mount_dir = os.path.abspath(mount_dir)
-        mnt_args = [
-            "--mount",
-            f"type=bind,source={mount_dir},destination=/cf_feedstock_ops_dir",
-        ]
-        if mount_readonly:
-            mnt_args[-1] += ",readonly"
-    else:
-        mnt_args = []
-
     extra_container_args = extra_container_args or []
 
     cmd = [
         *get_default_container_run_args(tmpfs_size_mb=tmpfs_size_mb),
-        *mnt_args,
         *_get_proxy_mode_container_args(),
         *extra_container_args,
         get_default_container_name(),
         *args,
     ]
-    res = subprocess.run(
-        cmd,
-        stdout=subprocess.PIPE,
-        text=True,
-        input=input,
-    )
```

Comment:
breaking change for library consumers that use a custom Docker image (i.e., cf-scripts):
stdin input can no longer be passed manually to the Docker container, virtual mounts have to be used (for feedstock attrs). 

`unpack_virtual_mounts_from_stdin` and `repack_virtual_mounts_to_stdout` have to be used inside the container to pack and unpack the tar from stdin and stdout